x/incentives/keeper/genesis_test.go

```diff
@@ -9,12 +9,80 @@ import (
 	"github.com/stretchr/testify/require"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 
+	"github.com/osmosis-labs/osmosis/osmomath"
 	osmoapp "github.com/osmosis-labs/osmosis/v19/app"
 
+	"github.com/osmosis-labs/osmosis/v19/x/concentrated-liquidity/model"
 	"github.com/osmosis-labs/osmosis/v19/x/incentives/types"
 	lockuptypes "github.com/osmosis-labs/osmosis/v19/x/lockup/types"
 )
 
+var (
+	distrToByDuration = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.ByDuration,
+		Denom:         "lptoken",
+		Duration:      time.Second,
+	}
+
+	distrToNoLock = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.NoLock,
+	}
+
+	distrToNoLockPool1 = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.NoLock,
+		Denom:         "no-lock/i/1",
+		Duration:      time.Hour * 24 * 7,
+		Timestamp:     time.Time{}.UTC(),
+	}
+
+	distrToNoLockPool2 = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.NoLock,
+		Denom:         "no-lock/i/2",
+		Duration:      time.Hour * 24 * 7,
+		Timestamp:     time.Time{}.UTC(),
+	}
+
+	distrToByGroup = lockuptypes.QueryCondition{LockQueryType: lockuptypes.ByGroup}
+
+	gaugeCoins = sdk.Coins{sdk.NewInt64Coin("stake", 10000)}
+
+	gaugeOneRecord = types.InternalGaugeRecord{
+		GaugeId:          1,
+		CurrentWeight:    osmomath.NewInt(100),
+		CumulativeWeight: osmomath.NewInt(200),
+	}
+
+	gaugeTwoRecord = types.InternalGaugeRecord{
+		GaugeId:          2,
+		CurrentWeight:    osmomath.NewInt(100),
+		CumulativeWeight: osmomath.NewInt(200),
+	}
+
+	expectedGroups = []types.Group{
+		{
+			GroupGaugeId: 5,
+			InternalGaugeInfo: types.InternalGaugeInfo{
+				TotalWeight:  gaugeOneRecord.CurrentWeight.Add(gaugeTwoRecord.CurrentWeight),
+				GaugeRecords: []types.InternalGaugeRecord{gaugeOneRecord, gaugeTwoRecord},
+			},
+			SplittingPolicy: types.ByVolume,
+		},
+	}
```

Comment:
nit: we should already have these defined in other files within the package. Can we reuse the existing variables to avoid repeated test variables that are essentially the same?

Reply:
The problem is the gauge ID of the second one is different, so if I use the old one and change the gauge ID in the tests themselves, I think it will end up being about the same number of lines.

Reply:
I think there is a helper `withGaugeID` that might help to address this

Reply:
Sadly only works on Gauge types, not InternalGaugeTypes